pkgs/servers/akkoma/pleroma-fe/default.nix

```diff
@@ -0,0 +1,85 @@
+{ lib
+, stdenv
+, fetchFromGitea, fetchYarnDeps
+, fixup_yarn_lock, yarn, nodejs
+, jpegoptim, oxipng, nodePackages
+}:
+
+stdenv.mkDerivation rec {
+  pname = "pleroma-fe";
+  version = "2022-09-10";
```

Comment:
```suggestion
  version = "unstable-2022-09-10";
```

Reply:
Although it is based on the stable branch which might create some confusion, it appears as that <https://nixos.org/manual/nixpkgs/unstable/#sec-package-naming> is quite strict on version names.